lightning/src/ln/channel.rs

```diff
@@ -6484,6 +6500,7 @@ impl<Signer: WriteableEcdsaChannelSigner> Writeable for Channel<Signer> {
 			(23, channel_ready_event_emitted, option),
 			(25, user_id_high_opt, option),
 			(27, self.channel_keys_id, required),
+			(29, requires_manual_readiness_signal, option)
```

Comment:
Interesting.  So to make sure I understand.  It's a user who had enabled this feature and got partially through the channel open process.  They then downgrade LDK to a version before this feature exists.  It means this TLV is persisted and the old version will just ignore it because it doesn't know about it.

In the downgraded ldk it will automatically signal channel ready without asking them.

I don't really think that's a problem? They explicitly downgraded to a version of LDK that no longer supports manual signaling?

If we make this even then it means when they try to start up the downgraded LDK it will panic when it tries to deserialize this type it doesn't know about?  

Basically forcing them to upgrade or wait for a bugfix for whatever reason they downgraded in the first place?

If all of this is correct then I guess I'm not so sure what is the better experience.  I guess it really just depends on _why_ they want to manually signal and partially why they are downgrading.